scripts/maintenance/migrate_project/README.md

```diff
@@ -1,34 +1,51 @@
 # project migration
 
 Built on top of the existing `postgres-database` package.
-It is used to migrate a user's project and eventually (hidden projects, generated via the api).
 
-If a file's or project's unique identifier already exist in the destination database, the process will not continue.
+It is used to migrate a user's project. Currently, this does not work for hidden projects, generated via the api.
 
-**NOTE:** data sync is done using `rclone`, currently it is assumed that the data source is a `MINIO S3 backend` and the destination is an `AWS S3 backend`.
+It uses low-level direct access to pgSQL and S3. Data sync is done using `rclone`. Therefore, this script might become outdated upon database changes in osparc-simcore.
 
 
-Any doubts? Ask **ANE**.
+# IMPORTANT PITFALLS:
+- Currently, this does not work for hidden projects, generated via the api.
+- No postgres database version migration is performed atm. This migration only works for identical source and target RDBs.
+- If a file's or project's UUID already exist in the destination database (colision), this script will fail with an error.
+- Supported S3 providers are `CEPH`, `AWS`, `MINIO`
```

Comment:
what about putting this *important* information in the help of the Makefile recipes?

Reply:
and maybe add a TIP on what should be done prior to using the script. I could expect that uploading a study that already exists might pretty much destroy the destination or does it stop before doing it?

Reply:
The Important Pitfalls sections partially handles this. will add a tip.